apps/charterafrica/src/pages/api/v1/resources/ecosystem/[action].page.js

```diff
@@ -0,0 +1,43 @@
+import { airtableSchema } from "@/charterafrica/lib/ecosystem/airtable";
+
+const isApiKeyValid = (key) => {
+  return key && key === process.env.RESOURCES_SECRET_TOKEN;
+};
+
+export async function schema(req, res) {
+  const { source } = req.query;
+  const sourceMap = { airtable: airtableSchema };
```

Comment:
Why do we need `airtable` as source? Is there any other schema we'll be exposing here?